en/components/grids_templates/selection.md

```diff
@@ -339,6 +339,99 @@ public selection = true;
 
 **Note:** Cell selection will trigger [`onSelection`]({environment:angularApiUrl}/classes/@@igTypeDoc.html#onselection) and not [`onRowSelectionChange`]({environment:angularApiUrl}/classes/@@igTypeDoc.html#onrowselectionchange).
 
+### Styling
+
+The theme engine exposes properties that allows us to style the range of selected cells.
+
+#### Import theme
+
+To get started with styling the selection, we need to import the `index` file, where all the theme functions and component mixins live:
+
+```scss
+// in component.scss
+@import '~igniteui-angular/lib/core/styles/themes/index';
+``` 
+
+#### Define colors
+
+Once done, we can make use of the [`igx-contrast-color`]({environment:sassApiUrl}/index.html#function-igx-contrast-color) and [`igx-color`]({environment:sassApiUrl}/index.html#function-igx-color) functions. With them, we define the colors we would like to use for our selection range:
+```scss
+    $text-color: igx-contrast-color($default-palette, 'primary', 900);
+    $background-color: igx-color($default-palette, "primary", 900);
+    $border-yellow: #f2c43c;
+```
+
+#### Create custom theme
+ 
+Next we create need to create a new theme that extends the [`igx-grid-theme`]({environment:sassApiUrl}/index.html#function-igx-grid-theme) passing our `text-color`, `background-color` and `border-yellow` variables as `$cell-selected-text-color`, `$cell-selected-background` and `$cell-active-border-color`, respectively:
+
+```scss
+$custom-grid-theme: igx-grid-paginator-theme(
+    $cell-selected-text-color: $text-color,
+    $cell-active-border-color: $border-yellow,
+    $cell-selected-background: $background-color
+);
+```
+
+#### Consume theme
+
+Afterwards, all we need to do is have our @@igSelector consume the newly created theme instead of the default one:
```

Comment:
The grid (`@@igSelector`) does the consuming of sorts, but what you are doing here is [including the theme mixin](https://sass-lang.com/documentation/at-rules/mixin) in the [app] component styles to override the default theme for the grid inside.

Reply:
Applies to the title as well. "Apply" would be more suitable.